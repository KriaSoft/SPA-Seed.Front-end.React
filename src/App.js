'use strict';

var React = require('react');

// Export React so the dev tools can find it
(window !== window.top ? window.top : window).React = React;

var Routes = require('./Routes.jsx')

React.renderComponent(Routes, document.body)
