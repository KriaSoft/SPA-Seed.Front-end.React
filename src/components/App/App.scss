@import '../../../node_modules/animate.css/animate.css';
@import '../../styles/base.scss';

.cover {
  position: absolute;
  left: 0;
  top: 0;
  right: 0;
  bottom: 0;
  background-color: white;
  z-index: -1;
  animation: cover 2.5s ease-in-out forwards;
}

.app {
  padding-left: 250px;
}

.content {
  opacity: 0;
  animation: fadeInUp 1s ease-in-out 5s forwards;
}
