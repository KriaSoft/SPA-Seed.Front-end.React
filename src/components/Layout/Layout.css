
html, body {
  height: 100%;
  -webkit-tap-highlight-color: transparent;
}

body{
  font-family: -apple-system-font, "Helvetica Neue", Helvetica, sans-serif;
}

ul{
  list-style: none;
}

body, .page {
  background-color: #F8F8F8;
}

.container {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  overflow: hidden;
}
