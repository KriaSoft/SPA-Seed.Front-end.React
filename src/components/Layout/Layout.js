/**
 * React Starter Kit (https://www.reactstarterkit.com/)
 *
 * Copyright © 2014-2016 Kriasoft, LLC. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */

import React, {PropTypes} from 'react';
import withStyles from 'isomorphic-style-loader/lib/withStyles';
import s from './Layout.css';
import WEUI from 'react-weui';
const {
  Tab,
  TabBody,
  TabBar,
  TabBarItem,
  TabBarLabel,
  TabBarIcon
} = WEUI;


class Layout extends React.Component {
  static propTypes = {
    children: PropTypes.node.isRequired,
  };

  render() {
    return (
      <div className={s.container}>
        <Tab>
          <TabBody>
            {this.props.children}
          </TabBody>
          <TabBar>
            <TabBarItem
              active={true}
              icon={<span className="iconfont icon-message"/>}
              label="微信"
            />
            <TabBarItem>
              <TabBarIcon>
                <span className="iconfont icon-contact"/>
              </TabBarIcon>
              <TabBarLabel>通讯录</TabBarLabel>
            </TabBarItem>
            <TabBarItem
              icon={<span className="iconfont icon-discover" />}
              label="发现"
            />
            <TabBarItem
              icon={<span className="iconfont icon-my" />}
              label="我"
            />
          </TabBar>
        </Tab>
      </div>
    );
  }
}

export default withStyles(s)(Layout);
