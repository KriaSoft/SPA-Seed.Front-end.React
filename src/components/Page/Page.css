/**
 * React Starter Kit (https://www.reactstarterkit.com/)
 *
 * Copyright Â© 2014-present Kriasoft, LLC. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */

@import '../variables.css';

.root {
  padding-left: 20px;
  padding-right: 20px;
}

.container {
  margin: 0 auto;
  padding: 0 0 40px;
  max-width: var(--max-content-width);

  &::after {
    content: '';
    clear: both;
    display: table;
  }
}

/* Grid fall-backs for <= IE 10 */
.sectionMainColumnWide,
.sectionMainColumnNarrow,
.sectionLeftColumn,
.sectionRightColumn {
  float: left;
}

.sectionMainColumnWide {
  width: var(--main-column-wide-width);
}

.sectionMainColumnNarrow {
  width: var(--main-column-narrow-width);
}

.sectionLeftColumn {
  width: var(--left-column-width);
}

.sectionRightColumn {
  width: var(--right-column-width);
}

@supports (display: grid) {
  .container {
    display: grid;

    &.leftColumn {
      grid-template-columns: 1fr 3fr;
    }

    &.rightColumn {
      grid-template-columns: 3fr 1fr;
    }

    &.multiColumn {
      grid-template-columns: 1fr 2fr 1fr;
    }
  }

  .sectionMainColumnWide,
  .sectionMainColumnNarrow,
  .sectionLeftColumn,
  .sectionRightColumn {
    width: unset;
  }
}
