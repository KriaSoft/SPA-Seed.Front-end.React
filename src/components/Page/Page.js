/**
 * React Starter Kit (https://www.reactstarterkit.com/)
 *
 * Copyright Â© 2014-present Kriasoft, LLC. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */

import React from 'react';
import PropTypes from 'prop-types';
import withStyles from 'isomorphic-style-loader/lib/withStyles';
import classNames from 'classnames';
import { titleize } from 'lodash-inflection';
// This is necessary because some classes are interpolated, not unused
// eslint-disable-next-line css-modules/no-unused-class
import s from './Page.css';

class Page extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired,
    html: PropTypes.string.isRequired,
    /* eslint-disable */
    leftColumnTitle: PropTypes.string,
    leftColumnHtml: PropTypes.string,
    rightColumnTitle: PropTypes.string,
    rightColumnHtml: PropTypes.string,
    /* eslint-enable */
  };
  static defaultProps = {
    leftColumnTitle: '',
    leftColumnHtml: '',
    rightColumnTitle: '',
    rightColumnHtml: '',
  };
  renderColumnTitle(side) {
    const titleKey = `${side}ColumnTitle`;
    if (!this.props[titleKey]) return '';
    return (
      <h2
        // eslint-disable-next-line react/no-danger
        dangerouslySetInnerHTML={{ __html: this.props[titleKey] }}
      />
    );
  }
  renderColumnHtml(side) {
    const columnHtml = this.props[`${side}ColumnHtml`];
    if (!columnHtml) return '';
    return (
      <section className={s[`section${titleize(side)}Column`]}>
        {this.renderColumnTitle(side)}
        <article
          // eslint-disable-next-line react/no-danger
          dangerouslySetInnerHTML={{ __html: columnHtml }}
        />
      </section>
    );
  }
  render() {
    const { title, html } = this.props;
    const hasAdjacentColumns =
      this.props.leftColumnHtml && this.props.rightColumnHtml;
    const hasOnlyLeftColumn = !hasAdjacentColumns && this.props.leftColumnHtml;
    const hasOnlyRightColumn =
      !hasAdjacentColumns && this.props.rightColumnHtml;
    return (
      <div className={s.root}>
        <div
          // The linter complains because these classes are nested
          /* eslint-disable css-modules/no-undef-class */
          className={classNames(s.container, {
            [s.multiColumn]: hasAdjacentColumns,
            [s.leftColumn]: hasOnlyLeftColumn,
            [s.rightColumn]: hasOnlyRightColumn,
          })}
          /* eslint-enable css-modules/no-undef-class */
        >
          {this.renderColumnHtml('left')}
          <section
            className={classNames({
              [s.sectionMainColumnNarrow]: hasAdjacentColumns,
              [s.sectionMainColumnWide]:
                hasOnlyRightColumn || hasOnlyLeftColumn,
            })}
          >
            <h1>{title}</h1>
            <div
              // eslint-disable-next-line react/no-danger
              dangerouslySetInnerHTML={{ __html: html }}
            />
          </section>
          {this.renderColumnHtml('right')}
        </div>
      </div>
    );
  }
}

export default withStyles(s)(Page);
