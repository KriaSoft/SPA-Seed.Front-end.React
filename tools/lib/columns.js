function getColumn(key, columnKey, columnsKey) {
  if (!columnKey) return {};
  let pageData = {};
  const routeKey = columnsKey || key;
  // eslint-disable-next-line import/no-dynamic-require,global-require
  const page = require(`../../src/routes/${routeKey}/${columnKey}.md`);
  if (columnKey.startsWith('left')) {
    const { title: leftColumnTitle, html: leftColumnHtml, key: colKey } = {
      ...page,
    };
    pageData = { leftColumnTitle, leftColumnHtml, colKey };
  } else {
    const { title: rightColumnTitle, html: rightColumnHtml, key: colKey } = {
      ...page,
    };
    pageData = { rightColumnTitle, rightColumnHtml, colKey };
  }
  return pageData;
}
const withColumns = (page, { columnsKey, leftColumnKey, rightColumnKey }) => {
  const key = columnsKey || page.key;
  Object.assign(
    page,
    getColumn(key, leftColumnKey, columnsKey),
    getColumn(key, rightColumnKey, columnsKey),
  );
};
export const defaultColumnData = {
  leftColumnKey: 'left-column',
  rightColumnKey: 'right-column',
  columnsKey: 'global-columns',
};
export default withColumns;
