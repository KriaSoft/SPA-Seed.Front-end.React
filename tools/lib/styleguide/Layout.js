import React, { Component } from 'react';
import PropTypes from 'prop-types';
import withStyles from 'isomorphic-style-loader/lib/withStyles';

// external-global styles must be imported in your JS.
import normalizeCss from 'normalize.css';
import layoutStyle from '../../../src/components/base/Layout/Layout.css';

/* Add the default Layout Styling to the Styleguide */
@withStyles(normalizeCss, layoutStyle)
class Layout extends Component {
  static propTypes = {
    children: PropTypes.node.isRequired,
  };

  render() {
    return <div>{this.props.children}</div>;
  }
}

export default Layout;
