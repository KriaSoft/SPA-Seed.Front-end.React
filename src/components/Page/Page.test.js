/**
 * React Starter Kit (https://www.reactstarterkit.com/)
 *
 * Copyright © 2014-present Kriasoft, LLC. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */

/* eslint-env jest */
/* eslint-disable padded-blocks, no-unused-expressions */

import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';
import Layout from '../Layout';
import Page from './Page';
import { defaultColumnData } from '../../../tools/lib/columns';

describe('Page', () => {
  let about = {};
  let columnData = {};
  beforeEach(() => {
    about = {
      title: 'About Us',
      component: 'ContentPage',
      key: 'about',
      html:
        '<p>Lorem ipsum dolor sit amet, consectetur adipisc…or arcu eget arcu. Vestibulum vel quam enim.</p>',
    };
    columnData = {
      leftColumnTitle: 'Whoa, a title!',
      leftColumnHtml: '<p>You will never escape me!</p>',
      rightColumnTitle: 'The correct column title',
      rightColumnHtml: '<ul><li>A list with one item</li></ul>',
    };
  });
  test('renders page without columns correctly', () => {
    const wrapper = renderer.create(
      <App context={{ insertCss: () => {}, fetch: () => {}, pathname: '' }}>
        <Layout>
          <Page {...about} />
        </Layout>
      </App>,
    );
    expect(wrapper).toMatchSnapshot();
  });
  test('renders multi column page correctly', () => {
    Object.assign(about, defaultColumnData, columnData);
    const wrapper = renderer.create(
      <App context={{ insertCss: () => {}, fetch: () => {}, pathname: '' }}>
        <Layout>
          <Page {...about} />
        </Layout>
      </App>,
    );
    expect(wrapper).toMatchSnapshot();
  });
  test('renders left column page correctly', () => {
    delete columnData.rightColumnHtml;
    delete columnData.rightColumnTitle;
    Object.assign(about, defaultColumnData, columnData);
    const wrapper = renderer.create(
      <App context={{ insertCss: () => {}, fetch: () => {}, pathname: '' }}>
        <Layout>
          <Page {...about} />
        </Layout>
      </App>,
    );
    expect(wrapper).toMatchSnapshot();
  });
  test('renders right column page correctly', () => {
    delete columnData.leftColumnHtml;
    delete columnData.leftColumnTitle;
    Object.assign(about, defaultColumnData, columnData);
    const wrapper = renderer.create(
      <App context={{ insertCss: () => {}, fetch: () => {}, pathname: '' }}>
        <Layout>
          <Page {...about} />
        </Layout>
      </App>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
