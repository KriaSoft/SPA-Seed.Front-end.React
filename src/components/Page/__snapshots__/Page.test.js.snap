// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders left column page correctly 1`] = `
<div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <div
        className="root"
        role="navigation"
      >
        <a
          className="link"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="link"
          href="/contact"
          onClick={[Function]}
        >
          Contact
        </a>
        <span
          className="spacer"
        >
           | 
        </span>
        <a
          className="link"
          href="/login"
          onClick={[Function]}
        >
          Log in
        </a>
        <span
          className="spacer"
        >
          or
        </span>
        <a
          className="link highlight"
          href="/register"
          onClick={[Function]}
        >
          Sign up
        </a>
      </div>
      <a
        className="brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="React"
          height="38"
          src="logo-small.png"
          srcSet="logo-small@2x.png 2x"
          width="38"
        />
        <span
          className="brandTxt"
        >
          Your Company
        </span>
      </a>
      <div
        className="banner"
      >
        <h1
          className="bannerTitle"
        >
          React
        </h1>
        <p
          className="bannerDesc"
        >
          Complex web apps made easy
        </p>
      </div>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container leftColumn"
    >
      <section
        className="sectionLeftColumn"
      >
        <h2
          dangerouslySetInnerHTML={
            Object {
              "__html": "Whoa, a title!",
            }
          }
        />
        <article
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>You will never escape me!</p>",
            }
          }
        />
      </section>
      <section
        className="sectionMainColumnWide"
      >
        <h1>
          About Us
        </h1>
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisc…or arcu eget arcu. Vestibulum vel quam enim.</p>",
            }
          }
        />
      </section>
      
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <a
        className="link"
        href="https://gitter.im/kriasoft/react-starter-kit"
      >
        Ask a question
      </a>
      <span
        className="spacer"
      >
        |
      </span>
      <a
        className="link"
        href="https://github.com/kriasoft/react-starter-kit/issues/new"
      >
        Report an issue
      </a>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <span
        className="text"
      >
        © Your Company
      </span>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/admin"
        onClick={[Function]}
      >
        Admin
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/privacy"
        onClick={[Function]}
      >
        Privacy
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/not-found"
        onClick={[Function]}
      >
        Not Found
      </a>
    </div>
  </div>
</div>
`;

exports[`Page renders multi column page correctly 1`] = `
<div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <div
        className="root"
        role="navigation"
      >
        <a
          className="link"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="link"
          href="/contact"
          onClick={[Function]}
        >
          Contact
        </a>
        <span
          className="spacer"
        >
           | 
        </span>
        <a
          className="link"
          href="/login"
          onClick={[Function]}
        >
          Log in
        </a>
        <span
          className="spacer"
        >
          or
        </span>
        <a
          className="link highlight"
          href="/register"
          onClick={[Function]}
        >
          Sign up
        </a>
      </div>
      <a
        className="brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="React"
          height="38"
          src="logo-small.png"
          srcSet="logo-small@2x.png 2x"
          width="38"
        />
        <span
          className="brandTxt"
        >
          Your Company
        </span>
      </a>
      <div
        className="banner"
      >
        <h1
          className="bannerTitle"
        >
          React
        </h1>
        <p
          className="bannerDesc"
        >
          Complex web apps made easy
        </p>
      </div>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container multiColumn"
    >
      <section
        className="sectionLeftColumn"
      >
        <h2
          dangerouslySetInnerHTML={
            Object {
              "__html": "Whoa, a title!",
            }
          }
        />
        <article
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>You will never escape me!</p>",
            }
          }
        />
      </section>
      <section
        className="sectionMainColumnNarrow"
      >
        <h1>
          About Us
        </h1>
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisc…or arcu eget arcu. Vestibulum vel quam enim.</p>",
            }
          }
        />
      </section>
      <section
        className="sectionRightColumn"
      >
        <h2
          dangerouslySetInnerHTML={
            Object {
              "__html": "The correct column title",
            }
          }
        />
        <article
          dangerouslySetInnerHTML={
            Object {
              "__html": "<ul><li>A list with one item</li></ul>",
            }
          }
        />
      </section>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <a
        className="link"
        href="https://gitter.im/kriasoft/react-starter-kit"
      >
        Ask a question
      </a>
      <span
        className="spacer"
      >
        |
      </span>
      <a
        className="link"
        href="https://github.com/kriasoft/react-starter-kit/issues/new"
      >
        Report an issue
      </a>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <span
        className="text"
      >
        © Your Company
      </span>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/admin"
        onClick={[Function]}
      >
        Admin
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/privacy"
        onClick={[Function]}
      >
        Privacy
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/not-found"
        onClick={[Function]}
      >
        Not Found
      </a>
    </div>
  </div>
</div>
`;

exports[`Page renders page without columns correctly 1`] = `
<div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <div
        className="root"
        role="navigation"
      >
        <a
          className="link"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="link"
          href="/contact"
          onClick={[Function]}
        >
          Contact
        </a>
        <span
          className="spacer"
        >
           | 
        </span>
        <a
          className="link"
          href="/login"
          onClick={[Function]}
        >
          Log in
        </a>
        <span
          className="spacer"
        >
          or
        </span>
        <a
          className="link highlight"
          href="/register"
          onClick={[Function]}
        >
          Sign up
        </a>
      </div>
      <a
        className="brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="React"
          height="38"
          src="logo-small.png"
          srcSet="logo-small@2x.png 2x"
          width="38"
        />
        <span
          className="brandTxt"
        >
          Your Company
        </span>
      </a>
      <div
        className="banner"
      >
        <h1
          className="bannerTitle"
        >
          React
        </h1>
        <p
          className="bannerDesc"
        >
          Complex web apps made easy
        </p>
      </div>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      
      <section
        className=""
      >
        <h1>
          About Us
        </h1>
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisc…or arcu eget arcu. Vestibulum vel quam enim.</p>",
            }
          }
        />
      </section>
      
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <a
        className="link"
        href="https://gitter.im/kriasoft/react-starter-kit"
      >
        Ask a question
      </a>
      <span
        className="spacer"
      >
        |
      </span>
      <a
        className="link"
        href="https://github.com/kriasoft/react-starter-kit/issues/new"
      >
        Report an issue
      </a>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <span
        className="text"
      >
        © Your Company
      </span>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/admin"
        onClick={[Function]}
      >
        Admin
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/privacy"
        onClick={[Function]}
      >
        Privacy
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/not-found"
        onClick={[Function]}
      >
        Not Found
      </a>
    </div>
  </div>
</div>
`;

exports[`Page renders right column page correctly 1`] = `
<div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <div
        className="root"
        role="navigation"
      >
        <a
          className="link"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="link"
          href="/contact"
          onClick={[Function]}
        >
          Contact
        </a>
        <span
          className="spacer"
        >
           | 
        </span>
        <a
          className="link"
          href="/login"
          onClick={[Function]}
        >
          Log in
        </a>
        <span
          className="spacer"
        >
          or
        </span>
        <a
          className="link highlight"
          href="/register"
          onClick={[Function]}
        >
          Sign up
        </a>
      </div>
      <a
        className="brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="React"
          height="38"
          src="logo-small.png"
          srcSet="logo-small@2x.png 2x"
          width="38"
        />
        <span
          className="brandTxt"
        >
          Your Company
        </span>
      </a>
      <div
        className="banner"
      >
        <h1
          className="bannerTitle"
        >
          React
        </h1>
        <p
          className="bannerDesc"
        >
          Complex web apps made easy
        </p>
      </div>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container rightColumn"
    >
      
      <section
        className="sectionMainColumnWide"
      >
        <h1>
          About Us
        </h1>
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisc…or arcu eget arcu. Vestibulum vel quam enim.</p>",
            }
          }
        />
      </section>
      <section
        className="sectionRightColumn"
      >
        <h2
          dangerouslySetInnerHTML={
            Object {
              "__html": "The correct column title",
            }
          }
        />
        <article
          dangerouslySetInnerHTML={
            Object {
              "__html": "<ul><li>A list with one item</li></ul>",
            }
          }
        />
      </section>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <a
        className="link"
        href="https://gitter.im/kriasoft/react-starter-kit"
      >
        Ask a question
      </a>
      <span
        className="spacer"
      >
        |
      </span>
      <a
        className="link"
        href="https://github.com/kriasoft/react-starter-kit/issues/new"
      >
        Report an issue
      </a>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="container"
    >
      <span
        className="text"
      >
        © Your Company
      </span>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/admin"
        onClick={[Function]}
      >
        Admin
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/privacy"
        onClick={[Function]}
      >
        Privacy
      </a>
      <span
        className="spacer"
      >
        ·
      </span>
      <a
        className="link"
        href="/not-found"
        onClick={[Function]}
      >
        Not Found
      </a>
    </div>
  </div>
</div>
`;
